packages/client/src/components/MovieCard/MovieCard.jsx

```diff
@@ -26,7 +26,9 @@ const BasicRating = () => {
 
 export const MovieCard = ({ movie }) => {
   const [favorites, toggleFavorite] = useFavorites([]);
-  const isFavorite = favorites.find((item) => item.id === movie.id);
+  const isFavorite = favorites.find(
+    (favoriteMovie) => favoriteMovie.id === movie.id,
```

Comment:
yes, it will work :) @itsgauravsaxena 
I have a problem pushing this file the first time, it doesn't show when I created PR so I decided to make changes in the name 
then after I push again, its updated in my PR 

I found out that one of my team accidentally remove this code while fixing other parts 
Original code was
const isFavorite = favorites.includes(movie) 
then I changed to favorites.find((item) => item.id === movie.id); 
then our team accidently remove this line of code 
then I create this PR again but because first push, doesnt show this code => make the changes with the name

Reply:
Ok makes sense then.

Reply:
Thank you so much!